### PR TITLE
Bumped Prettier from 1.16.4 to 1.18.0

### PR DESCRIPTION
**Pull request by andrew-demers**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "ember-router-generator": "^1.2.3",
     "inflection": "^1.12.0",
     "jquery": "^3.4.1",
+    "prettier": "^1.18.2",
```
Comment (rwjblue) [inline package.json]: Shouldn't be a dependency, can you move this back down to a devDep?

**Comment by rwjblue** [comment]
The update results in a few failures:

```
/home/travis/build/emberjs/ember.js/packages/@ember/-internals/container/lib/container.ts
  598:45  error  Replace `⏎··········this.normalizedName⏎········` with `this.normalizedName`  prettier/prettier
/home/travis/build/emberjs/ember.js/packages/@ember/-internals/glimmer/lib/modifiers/on.ts
  112:120  error  Replace `⏎········args.positional.length⏎······` with `args.positional.length`  prettier/prettier
/home/travis/build/emberjs/ember.js/packages/@ember/-internals/metal/lib/tracked.ts
  138:75  error  Replace `⏎··········propertyDesc.initializer⏎········` with `propertyDesc.initializer`  prettier/prettier
/home/travis/build/emberjs/ember.js/packages/@ember/-internals/routing/lib/system/route.ts
   256:23  error  Replace `⏎··········this.routeName⏎········` with `this.routeName`  prettier/prettier
  2367:89  error  Replace `⏎········this.routeName⏎······` with `this.routeName`      prettier/prettier
/home/travis/build/emberjs/ember.js/packages/@ember/-internals/routing/lib/system/router.ts
   516:75   error  Replace `⏎··········args[0]⏎········` with `args[0]`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        prettier/prettier
   525:43   error  Replace `⏎········this.currentRouteName⏎······` with `this.currentRouteName`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                prettier/prettier
   962:123  error  Replace `⏎················qpOther.scopedPropertyName⏎··············}\`·and·\`${⏎················qp.scopedPropertyName⏎··············}\`·map·to·\`${urlKey}\`.·You·can·fix·this·by·mapping·one·of·the·controller·properties·to·a·different·query·param·key·via·the·\`as\`·config·option,·e.g.·\`${⏎················qpOther.prop⏎··············` with `qpOther.scopedPropertyName}\`·and·\`${qp.scopedPropertyName}\`·map·to·\`${urlKey}\`.·You·can·fix·this·by·mapping·one·of·the·controller·properties·to·a·different·query·param·key·via·the·\`as\`·config·option,·e.g.·\`${qpOther.prop`  prettier/prettier
  1070:89   error  Replace `⏎············qp.route.routeName⏎··········` with `qp.route.routeName`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              prettier/prettier
/home/travis/build/emberjs/ember.js/packages/@ember/-internals/runtime/lib/system/core_object.js
  295:38  error  Replace `⏎········this.constructor⏎······` with `this.constructor`  prettier/prettier
/home/travis/build/emberjs/ember.js/packages/@ember/-internals/views/lib/system/jquery.js
  30:16  error  Replace `(jQueryDisabled·?·undefined·:·jQuery)` with `jQueryDisabled·?·undefined·:·jQuery`  prettier/prettier
/home/travis/build/emberjs/ember.js/packages/@ember/object/tests/computed/reduce_computed_macros_test.js
  1535:20  error  Insert `(`  prettier/prettier
  1541:10  error  Insert `)`  prettier/prettier
/home/travis/build/emberjs/ember.js/packages/internal-test-helpers/lib/ember-dev/method-call-tracker.ts
  164:143  error  Replace `⏎············actual[0]⏎··········` with `actual[0]`  prettier/prettier
  171:143  error  Replace `⏎············actual[0]⏎··········` with `actual[0]`  prettier/prettier
✖ 15 problems (15 errors, 0 warnings)
  15 errors and 0 warnings potentially fixable with the `--fix` option.
```

Can you run `yarn lint:fix`?

**Review by rwjblue — CHANGES_REQUESTED** [review]

**Comment by demersaj** [comment]
I fixed the linter and moved Prettier back to a devDep.

**Review by rwjblue — APPROVED** [review]
Thank you!

**Pull request merged**